packages/salesforce/src/Adaptor.js

```diff
@@ -783,6 +783,56 @@ export function toUTF8(input) {
   return anyAscii(input);
 }
 
+/**
```

Comment:
I don't like the salesforce docs. Try this:
``````
/**
 * Send a HTTP request using connected session information.
 *
 * @example
 * request('/actions/custom/flow/POC_OpenFN_Test_Flow', {
 *   method: 'POST',
 *   body: { inputs: [{}] },
 * });
 * @param {String} url - Relative or absolute URL to request from
 * @param {Object} options - Request options
 * @param {String} options.method - HTTP method to use. Defaults to GET
 * @param {Object} [options.headers] - Object of request headers
 * @param {Object} [options.body] - Request body
 * @param {Function} callback - A callback to execute once the request is complete
 * @returns {Operation}
 */
 ```